src/Common/src/TypeSystem/IL/Stubs/TypeSystemThrowingILEmitter.cs

```diff
@@ -61,7 +61,7 @@ public static MethodIL EmitIL(MethodDesc methodThatShouldThrow, TypeSystemExcept
 
             foreach (var arg in exception.Arguments)
             {
-                codeStream.Emit(ILOpcode.ldstr, emitter.NewToken(arg));
+                codeStream.Emit(ILOpcode.ldstr, emitter.NewToken(arg ?? ""));
```

Comment:
If you're not fixing this, please file an issue on me and add a link to it here.